backend/clap/proto/data_store.proto

```diff
@@ -151,6 +151,23 @@ message SubmitRatingDraft {
     int32 rating_id = 2;
 }
 
+message Answer {
+    string description = 1;
+    repeated string files = 2;
+}
+
+message Question {
+    repeated Answer answers = 1;
+}
+
+message StudyVisit {
+    repeated Question questions = 1;
+}
+
+message StudyVisitResponse {
+    repeated StudyVisit visits = 1;
+}
```

Comment:
There can be only one visit (list of question/answers) associated with a specific submission, so I think `repeated` is not needed here.